### PR TITLE
docs: Clarify wording on add_newline option

### LINKED ISSUE #1987: Clarify what `add_newline` and `line_break` do
#### Is your feature request related to a problem? Please describe.

We've had a number of questions about how disabling `add_newline` wasn't working when what the user really wanted to do was disable the `line_break` module. The most recently issue is #1982, but there are other issues, discussions, and Discord questions around the same subject.

#### Describe the solution you'd like

Clarify in the documentation that `add_newline` is not responsible for splitting the prompt into two lines.

I don't know what the best way to do this is: adding a lot of text to the description of `add_newline` on the configuration page seems like it could add noise to our docs. An FAQ entry might be nice, but I don't think many people will look at the FAQ page if they think that `add_newline` does what they want.

Maybe an FAQ entry with a link and short notice on the config page would be good? I'm also open to other ways of formatting this,

#### Describe alternatives you've considered

Changing the name of the option, but I think that ship has sailed.

### PR DESCRIPTION
**Pull request by adaschevici**
#### Description
Change wording for the confusing `add_newline` sentence to something that hopefully makes more sense.

#### Motivation and Context
It aims to prevent users from being confused by the subtle difference between `add_newline` and `line_break` options

Closes #1987 

#### How Has This Been Tested?
Tested by checking the docs local build.

- [x] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: docs/config/README.md
Comment (chipbuster) [inline docs/config/README.md:12]: Do you think clarifying that this is  a "blank line" instead of newline will help?

To me, a newline is just a line break, whereas a "blank line" is specifically a whole line with nothing on it--perhaps this will help further clear up some of the confusion? (It doesn't make much sense to insert a blank line in the middle of the prompt).

**Review by chipbuster — COMMENTED** [review]
Just a suggestion. If you think "newline" is better, then we should be good to merge this as-is.

File: docs/config/README.md
Comment (adaschevici) [reply docs/config/README.md]: makes sense, updating

**Review by chipbuster — APPROVED** [review]
LGTM but I'd like to get approval from at least one other astronaut before merging.

**Review by cappyzawa — APPROVED** [review]
LGTM

**Comment by andytom** [comment]
Thank you for your contribution @adaschevici.

**Pull request merged**
